README.md

```diff
@@ -2,12 +2,15 @@
 
 Utilities for modeling SCTE 224 data in Go
 
-Details on SCTE 224 can be found at: https://www.scte.org/SCTE/Standards/ANSI_SCTE_224_Event_Scheduling_and_Notification_Interface.aspx
+Details on SCTE 224 can be found at: https://www.scte.org/standards-development/library/standards-catalog/ansiscte-224-2018r1/
 
-The most recent spec: https://www.scte.org/SCTEDocs/Standards/ANSI_SCTE%20224%202018r1.pdf
+2015 schema: http://www.scte.org/schemas/224/SCTE224-20151115.xsd (link no longer working, refer to the spec located alongside structs in this repo)
 
-2015 schema: http://www.scte.org/schemas/224/SCTE224-20151115.xsd
+2018 schema: http://www.scte.org/schemas/224/SCTE224-20180501.xsd (link no longer working, refer to the spec located alongside structs in this repo)
 
-2018 schema: http://www.scte.org/schemas/224/SCTE224-20180501.xsd
+Latest schema (currently points to 2020): https://www.scte.org/standards-development/library/standards-catalog/ansiscte-224-2018r1/
```

Comment:
wow, super weird that the latest schema link https://scte-cms-resource-storage.s3.amazonaws.com/SCTE_224_20200407-1600425639358.xsd is found on a page referencing the 2021 schema located on a url mentioning 2018r1 @skurkows seems like SCTE still has wacky page structures